File: app/src/main/java/me/devsaki/hentoid/ui/BlinkAnimation.java
package me.devsaki.hentoid.ui;

import android.view.animation.AlphaAnimation;
import android.view.animation.Animation;

public class BlinkAnimation extends AlphaAnimation {

    public BlinkAnimation(long duration, long startOffset) {
        super(0.0f, 1.0f);
        setRepeatMode(Animation.REVERSE);
        setRepeatCount(Animation.INFINITE);
        setDuration(duration);
        setStartOffset(startOffset);
    }
}

File: app/src/main/java/me/devsaki/hentoid/ui/BlinkAnimation.kt
package me.devsaki.hentoid.ui

import android.view.animation.AlphaAnimation

class BlinkAnimation(duration: Long, startOffset: Long) : AlphaAnimation(0.0f, 1.0f) {
    init {
        repeatMode = REVERSE
        repeatCount = INFINITE
        setDuration(duration)
        setStartOffset(startOffset)
    }
}